app/src/component/transactionList/internal/transaction-list-item.js

```diff
@@ -11,14 +12,20 @@ class TransactionListItem extends React.PureComponent {
   }
   
   render() {
-    const { transaction } = this.props;
+    const { transaction, navigation } = this.props;
     const { amount, claim_id: claimId, claim_name: name, date, fee, txid, type } = transaction;
-    
+            
     return (
       <View style={transactionListStyle.listItem}>
         <View style={[transactionListStyle.row, transactionListStyle.topRow]}>
           <View style={transactionListStyle.col}>
             <Text style={transactionListStyle.text}>{this.capitalize(type)}</Text>
+            {name && claimId && (
+              <Link
```

Comment:
Will this try opening the browser? 

```
+    if (navigation && href.startsWith('#')) {
+      navigation.navigate(href.substring(1));
+    } else {
+      Linking.openURL(href).catch(err => notify({
+        message: error,
+        displayType: ['toast']
+      }));
+    }
```

I think it will go into the `else` block and try opening the browser with not href.

Reply:
I have the `onPress` specified explicitly for this link, which overrides the `onPress` in the component itself. Basically, the idea is you can override the default implemented Link opening behaviour if you want to implement some custom logic.

Reply:
Ah I see that now. Cool